crates/sui/src/ptb/ptb_parser/build_ptb.rs

```diff
@@ -275,38 +262,38 @@ impl GasBudget {
         }
 
         let budget = if self.gas_budgets.len() == 1 {
-            self.gas_budgets[0].value.1
+            self.gas_budgets[0].value
         } else {
-            match self.picker.get(0).map(|x| &x.value.1) {
+            match self.picker.get(0).map(|x| &x.value) {
                 Some(GasPicker::Max) => self
                     .gas_budgets
                     .iter()
-                    .map(|x| x.value.1)
+                    .map(|x| x.value)
                     .max()
                     .unwrap()
```

Comment:
This should be safe because we verified that `gas_budgets` is non-empty above. So unwrapping here should be fine (unless someone provides gas budgets that wrap in their sum, but that will cause a different error).